docs/tutorials/zero-to-hero.md

```diff
@@ -41,130 +32,35 @@ Let's get started!
 >
 > You can interact with example apps online, or explore the code by opening it in gitpod (online IDE). To do that, click on the example you want to see, and then click "Open in gitpod". Gitpod online IDE will open with the project loaded.
 
-All our sample projects work out of the box. Pick any assemblyscript project to proceed (for example, guest book). The project file structure is explained in more detail [here](/docs/quick-start/development-overview)
+All our sample projects work out of the box. Pick the  (for example, guest book). The project file structure is explained in more detail [here](/docs/quick-start/development-overview)
```

Comment:
let's not give them the choice (remove "for example, guest book"), and tell them to use Wallet (see comment below)
https://github.com/near-examples/wallet-example

Reply:
Fixed